src/static/css/Location.css

```diff
@@ -0,0 +1,103 @@
+.location-background {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  min-height: 100vh;
+  background: #ffffff;
+  display: flex;
+  flex-direction: row;
+  z-index: 1;
+  display: flex;
+}
+
+.split {
+  height: 100%;
+  width: 50%;
+  /* position: fixed; */
+  top: 0;
+  overflow-x: hidden;
+}
+
+.left {
+  left: 0;
```

Comment:
The left and right need `position: relative`, otherwise the images appear at the top of the slide deck.